### PR TITLE
Tickets/dm 33788

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

**Review by r-owen — APPROVED** [review]
A few minor suggestion. Overall it looks fine.

File: python/lsst/ts/ess/controller/command_handler.py [hunk 1]
```diff
@@ -122,24 +120,15 @@ def create_device(
...
+                f"and sensor {sensor}"
+            )
+            device = RpiSerialHat(
+                name=device_configuration[common.Key.NAME],
+                device_id=device_configuration[common.Key.SERIAL_PORT],
+                sensor=sensor,
+                callback_func=self._callback,
+                log=self.log,
+            )
+            return device
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/ess/controller/command_handler.py:134]: Thanks for catching that. Those lines were deleted in error and I restored them.

File: python/lsst/ts/ess/controller/device/rpi_serial_hat.py [hunk 2]
```diff
@@ -67,6 +67,8 @@ def __init__(
             self.log.exception(e)
             # Unrecoverable error, so propagate error
             raise e
+        # get event loop to run blocking tasks
+        self.loop = asyncio.get_event_loop()
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/ess/controller/device/rpi_serial_hat.py:71]: I switched to using `get_running_loop` and moved the lines to get the loop back to where the loop is needed.

File: python/lsst/ts/ess/controller/device/vcp_ftdi.py [hunk 3]
```diff
@@ -67,6 +68,8 @@ def __init__(
             lazy_open=True,
             auto_detach=False,
         )
+        # get event loop to run blocking tasks
+        self.loop = asyncio.get_event_loop()
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/ess/controller/device/vcp_ftdi.py:72]: Done.

**Pull request merged**